site/content/en/docs/enterprise/social-accounts-configuration.md

```diff
@@ -106,32 +106,61 @@ There are 2 basic steps to enable GitHub account authentication.
 > but don't forget to add required permissions.
 > <br>In the **Permission** > **Account permissions** > **Email addresses** must be set to **read-only**.
 
-## Enable authentication with an Amazon Cognito
-
-To enable authentication, do the following:
-
-1. Create a user pool. For more information,
-   see [Amazon Cognito user pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html)
-2. Fill in the name field, set the homepage URL (for example: `https://localhost:8080`),
-   and authentication callback URL (for example: `https://localhost:8080/api/auth/social/amazon-cognito/login/callback/`).
-3. Create conпiguration file in CVAT:
-
-   1. Create the `auth_config.yml` file with the following content:
+## Enable Authentication with Amazon Cognito
+
+To enable authentication with Amazon Cognito for your CVAT instance, you need to complete two main steps:
+
+1. **Create and configure an [Amazon Cognito user pool](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html)**.
+   Follow these detailed steps to set up the user pool:
+   - Create a user pool. For example, you might use settings like those shown in the image below:
+     ![](/images/cognito_pool_1.png)
+
+   - Configure a new app on the step `Integrate your app`:
+     - Select the `Confidential client` type, as CVAT securely stores
+     client secrets on the server side.
+       (_Note_: the `Public client` type is also supported.)
+     - Enter a name for your app client.
+     - Choose the `Generate a client secret` option.
+       ![](/images/cognito_pool_2.png)
+
+   - Once your pool is configured, go to the `App integration` tab
+   on the pool details page and then to the `Domain` section.
+     Create either a custom domain or Cognito domain (e.g., `https://cvat.auth.us-east-1.amazoncognito.com`).
```

Comment:
`cvat` is probably not the best suggestion here, as a) we shouldn't assume the pool is just for CVAT (if it is, why use Cognito at all?), and b) the name has to be unique within the region, so different organizations would need to use different names. I would replace `cvat` with a placeholder like `<my_org>`. Or just stick with `custom-cognito-prefix`, like in the config below.

Reply:
removed this part